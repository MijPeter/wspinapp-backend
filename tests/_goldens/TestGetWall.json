{
    "ID": 4,
    "CreatedAt": "2006-01-02T15:04:05+01:00",
    "UpdatedAt": "2006-01-02T15:04:05+01:00",
    "DeletedAt": null,
    "Holds": [
        {
            "ID": 7,
            "CreatedAt": "2006-01-02T15:04:05+01:00",
            "UpdatedAt": "2006-01-02T15:04:05+01:00",
            "DeletedAt": null,
            "X": 120.03,
            "Y": 256.43,
            "Size": 0,
            "WallID": 4,
            "Shape": "Circle",
            "Angle": 0
        },
        {
            "ID": 8,
            "CreatedAt": "2006-01-02T15:04:05+01:00",
            "UpdatedAt": "2006-01-02T15:04:05+01:00",
            "DeletedAt": null,
            "X": 120.03,
            "Y": 36.43,
            "Size": 0,
            "WallID": 4,
            "Shape": "circle",
            "Angle": 0
        }
    ],
    "ImageUrl": "",
    "ImagePreviewUrl": ""
}