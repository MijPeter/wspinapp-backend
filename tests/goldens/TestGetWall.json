{
    "ID": 4,
    "CreatedAt": "2023-01-01T10:00:00+01:00",
    "UpdatedAt": "2023-01-01T10:00:00+01:00",
    "DeletedAt": null,
    "Holds": [
        {
            "ID": 7,
            "CreatedAt": "2023-01-01T10:00:00+01:00",
            "UpdatedAt": "2023-01-01T10:00:00+01:00",
            "DeletedAt": null,
            "X": 120.03,
            "Y": 256.43,
            "Size": 0,
            "WallID": 4,
            "Shape": "Circle",
            "Angle": 0
        },
        {
            "ID": 8,
            "CreatedAt": "2023-01-01T10:00:00+01:00",
            "UpdatedAt": "2023-01-01T10:00:00+01:00",
            "DeletedAt": null,
            "X": 120.03,
            "Y": 36.43,
            "Size": 0,
            "WallID": 4,
            "Shape": "circle",
            "Angle": 0
        }
    ],
    "ImageUrl": "",
    "ImagePreviewUrl": ""
}